sql/search/search_rfm/query.sql

```diff
@@ -1,6 +1,3 @@
-CREATE OR REPLACE VIEW
-  `moz-fx-data-shared-prod.search.search_rfm`
-AS
 SELECT
   `moz-fx-data-shared-prod.udf.days_seen_bytes_to_rfm`(days_seen_bytes) AS days_seen,
```

Comment:
This query should use relative UDF and table references, because fully qualified references are only needed for views:
```suggestion
  udf.days_seen_bytes_to_rfm(days_seen_bytes) AS days_seen,
```